README.md

```diff
@@ -1,9 +1,55 @@
 [![Build Status](https://travis-ci.org/TREP-LABS/realdrip-backend.svg?branch=master)](https://travis-ci.org/TREP-LABS/realdrip-backend)
 [![Coverage Status](https://coveralls.io/repos/github/TREP-LABS/realdrip-backend/badge.svg?branch=master)](https://coveralls.io/github/TREP-LABS/realdrip-backend?branch=master)
-## RealDrip Backend
-`A node-js server using express-js
-`
+
+![GitHub contributors](https://img.shields.io/github/contributors/trep-labs/realdrip-backend)
+![GitHub commit activity](https://img.shields.io/github/commit-activity/m/trep-labs/realdrip-backend)
+![GitHub pull requests](https://img.shields.io/github/issues-pr/trep-labs/realdrip-backend)
+![GitHub closed pull requests](https://img.shields.io/github/issues-pr-closed-raw/trep-labs/realdrip-backend)
+# RealDrip Backend
+ > [Learn about The Realdrip Device](https://treplabs.co/realdrip)
+
 #### The Setup integrated the following CI/CD tools
  - TravisCI
  - Coveralls.io
- - Jest
\ No newline at end of file
+ - Jest & Suppertest
+ #### Technologies
+ - Nodejs
+ - MongoDB
+ - Javascript
+
+## Important Dependacies
+ - [mongoose](https://mongoosejs.com/)
+ - [Jsonwebtoken](https://www.npmjs.com/package/jsonwebtoken)
+ - [validator](https://www.npmjs.com/package/validator)
+ - [nodemailer](https://nodemailer.com/)
+ - [expressJs](https://expressjs.com/)
+
+## Project setup
+```
+git clone https://github.com/TREP-LABS/realdrip-backend.git
+npm install
+```
```

Comment:
If I was to give you just these two instructions, would it be enough to help you have the project up and running on your machine?
I believe you can do better on documenting the project setup, you can check [this](https://github.com/IAMOTZ/ievents#getting-started) for example.

Reply:
okay. i'll work on that